src/app/components/page/landing-page/landing-page.component.html

```diff
@@ -0,0 +1,88 @@
+<div class="page landing_page">
+  <section class="section_gradient section_full2">
+    <h2 class="left nopadding">What is</h2>
+    <h1 class="left">PhyloDiver?</h1>
+
+    <div class="columns">
+      <div class="column intro">
+        <p>PhyloDiver is a interactive tool for visualizing and exploring evolutionary phylogenies constructed from analyses of genetic variations in tumor samples. This tool was designed to simultaneously enable end-users to examine mutation histories while maintaining sample proportions of different <span class="nowrap">subclone populations.</span></p>
+        <p>PhyloDiver�s unique visualizations also connect to the underlying annotated mutations allowing for a comprehensive examination of the history, composition, and proportion of clonal subpopulations in <span class="nowrap">cancer tumors.</span></p>
+      </div>
+
+      <div class="column intro_diagram">
+        <img src="assets/images/landing_screenshot_001.png" />
+      </div>
+    </div>
+  </section>
+
+  <section class="section_padded">
+    <h2>Getting Started</h2>
+
+    <div class="block_list">
+      <div class="block">
+        <div class="block_title">Explore the Demo</div>
+
+        <div class="block_content">
+          <p>The PhyloDiver demo visualizes example data from an ultra-deep (300x-400x) bulk sequencing dataset containing the somatic mutations of a primary acute myeloid leukemia (AML) and its <span class="nowrap">subsequent relapse.</span></p>
+          <p>M. Griffith, et al. "Optimizing cancer genome sequencing and analysis." <span class="nowrap">Cell systems. 2015</span></p>
+        </div>
+
+        <div class="block_button">Take Me to the Demo!</div>
+      </div>
+
+      <div class="block">
+        <div class="block_title">Visualize Your Data</div>
+
+        <div class="block_content">
+          <p>Upload your JSON file in the format <span class="nowrap">specified here:</span><p>
+          <p>
+            <span class="link">Help</span>
+          </p>
+          <p>Data will never be travelin on <span class="nowrap">the internet.</span></p>
```

Comment:
I think this was placeholder text. I should probably just make a ticket for finalizing this content, unless you, Lisa, and/or Charles already talked about it and arrived at other plans.